test/unit/annotations/__snapshots__/deprecated.spec.ts.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`extension deprecation: code 1`] = `""`;
+
+exports[`extension deprecation: code 2`] = `"local __meta__A = debug.getregistry().A"`;
+
+exports[`extension deprecation: diagnostics 1`] = `"main.ts(4,9): warning TSTL: '@extension' is deprecated and will be removed in a future update. Please update your code before upgrading to the next release, otherwise your project will no longer compile. See https://typescripttolua.github.io/docs/advanced/compiler-annotations#extension for more information."`;
+
+exports[`extension deprecation: diagnostics 2`] = `"main.ts(4,9): warning TSTL: '@metaExtension' is deprecated and will be removed in a future update. Please update your code before upgrading to the next release, otherwise your project will no longer compile. See https://typescripttolua.github.io/docs/advanced/compiler-annotations#metaExtension for more information."`;
```

Comment:
think link doesn't work, I think the kind needs to be lowercased

Reply:
lowercased it, seems to work now. Also added https://github.com/TypeScriptToLua/TypeScriptToLua.github.io/pull/19 (WIP)